src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => {
+          const isTabAction = tab.id === selectedTab.id;
```

Comment:
```suggestion
          const isTabActive = tab.id === selectedTab.id;
```
Maybe `isTabActive`?) Because I don't get what is tab action